setup.sh

```diff
@@ -70,6 +70,7 @@ pssh --inline \
     --host "$MASTERS $SLAVES" \
     --user root \
     --extra-args "-t -t $SSH_OPTS" \
+    --timeout 300 \
```

Comment:
Actually does setting it to 0 match our previous `ssh` loop ? If so, it should be fine to set it to 0.